website/docs/api/admin/feature-toggles-api.md

```diff
@@ -283,7 +335,12 @@ None
 }
 ```
 
-### Mark a Feature Toggle as "stale" {#mark-a-feature-toggle-as-stale}
+## Mark a Feature Toggle as "stale" {#mark-a-feature-toggle-as-stale}
+
+:::caution Deprecation notice
+This endpoint is deprecated. Please use the [project-based endpoint to patch a feature toggle and mark it as stale](./feature-toggles-api-v2.md#patch-toggle) instead.
```

Comment:
Is this the right place to go for updating staleness?

Reply:
Yeah, looks like `PATCH` will update staleness.